docs/toc.md

```diff
@@ -127,7 +127,20 @@
 
 <!-- .NET Core Content -->
 # [.NET Core Guide](core/index.md)
-## [Getting started](core/getting-started.md)
+## [Getting started](core/getting-started/index.md)
+### [Install](core/getting-started/install/index.md)
+#### [Visual Studio 2017](win-vs2017.md)
+#### [Visual Studio 2015 Update 3](win-vs2015.md)
+#### [Windows with CLI Tools (VSCode)](win-cli.md)
+#### [RHEL](rhel.md)
```

Comment:
these links are all broken. take a look at the build report. they should look like core/getting-started/install/rhel.md